app/components/provider_interface/application_choice_header_component.html.erb

```diff
@@ -24,7 +24,7 @@
               <%= govuk_button_link_to 'Set up interview', new_provider_interface_application_choice_interview_path(application_choice), class: 'govuk-!-margin-bottom-0' %>
             <% end %>
             <% if respond_to_application? %>
-              <%= govuk_button_link_to 'Make decision', new_provider_interface_application_choice_decision_path(application_choice), secondary: set_up_interview?, class: 'govuk-!-margin-bottom-0' %>
+              <%= govuk_button_link_to 'Make decision', new_provider_interface_application_choice_decision_path(application_choice), class: "govuk-!-margin-bottom-0#{" govuk-button--secondary" if set_up_interview?}" %>
```

Comment:
could we move this to the component class itself as `def classes` or something similar to allow easier testing

Reply:
Done!